azureiot.sln

```diff
@@ -69,6 +69,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Sha
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "E2ETests", "e2e\test\E2ETests.csproj", "{8E25CDE3-992D-4942-8C38-51A0D8E8EB70}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DeviceClientAmqpMultiplexingSample", "iothub\device\samples\DeviceClientAmqpMultiplexingSample\DeviceClientAmqpMultiplexingSample.csproj", "{4FF8AA54-CB2F-45CE-BE68-E7669361D868}"
```

Comment:
>DeviceClientAmqpMultiplexingSample [](start = 53, length = 34)

samples for C# has move to sample repo...

Reply:
Yes, I just see it in the CR

---
In reply to: [251623281](https://github.com/Azure/azure-iot-sdk-csharp/pull/770#discussion_r251623281) [](ancestors = 251623281)